src/main/java/meowmeow/Meowmeow.java

```diff
@@ -0,0 +1,51 @@
+package meowmeow;
+import meowmeow.commands.Command;
+
+/**
+ * <p>The Meowmeow class is the main class of the program.</p>
+ * <p>This class is used to run the program.</p>
+ */
+public class Meowmeow {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the Meowmeow class.
+     * @param filePath is a String that specifies the filepath of the save file.
```

Comment:
Shouldn't there be an empty line between the description and parameter sections in Javadoc comments?
```suggestion
     * Constructor for the Meowmeow class.
     *
     * @param filePath is a String that specifies the filepath of the save file.
```